content/docs/introduction/roadmap.md

```diff
@@ -1,68 +1,93 @@
 ---
 
-Our developers are focused on making Neon the default choice for serverless Postgres. This roadmap describes committed features that are coming soon. We are as excited as you are to see new features in Neon, but their development, release, and timing are at our discretion.
+Our development teams are focused on helping you ship faster with Postgres. This roadmap describes committed features that we're working on right now, plus a peak at some of the upcoming features we'll be taking on next.
 
-As always, we are listening. If you see something you like, something you disagree with, or something you'd love for us to add, let us know in our Discord feedback channel.
+## What we've just launched
 
-<CommunityBanner buttonText="Leave feedback" buttonUrl="https://discord.com/channels/1176467419317940276/1176788564890112042" logo="discord">Share your ideas in&nbsp;Discord</CommunityBanner>
+For the latest features and fixes, check our [Changelog](/docs/changelog), updated every Friday. Or watch for our Changelog email, also sent regularly on Fridays. You can also subscribe to updates using our [RSS feed](https://neon.tech/docs/changelog/rss.xml).
 
-For the latest features and fixes, check our [Changelog](/docs/changelog), updated every Friday. You can subscribe to updates using our [RSS feed](https://neon.tech/docs/changelog/rss.xml).
+## What's around the corner
 
-## Neon Early Access Program
+Here's a snapshot of what we're working on now:
 
-If you would like to get a little more involved, consider signing up for the **Neon Early Access Program**.
+### Neon on Azure
 
-Benefits of joining:
+If you didn't catch the post, Neon is coming to Azure. [Read more](https://neon.tech/blog/neon-is-coming-to-azure) about this big news.
 
-- **Exclusive early access**: Get a first look at upcoming features before they go live.
-- **Private community**: Gain access to a dedicated Discord channel to connect with the Neon team and provide feedback to help shape what comes next.
-- **Weekly insights**: Receive updates on Neon's latest developments and future plans.
+### Database-as-a-Service
 
-[Sign Up Now](https://neon.tech/early-access-program) and start influencing the future of Neon!
+- **Autoscaling GA**: One of our defining features, Autoscaling is soon going GA. Look for an announcement with details coming out soon!
+- **Autoscaling on the Free Plan**: Users on the Free Plan will soon be able to try Neon's Autoscaling feature, within reasonable Free Plan limits.
+- **Database deploy previews**: Our GitHub App is available now for all users, with more refinements to come &#8212; including better integration with GitHub Actions, making it easier to incorporate your database into your development workflow.
+- **Better deletes**: We're adding support for deleting obsolete branches, especially after [restore](/docs/guides/branch-restore) operations.
 
-## Coming soon
+  If you have other branch management ideas, [let us know](#share-your-thoughts).
```

Comment:
Maybe make this a general statement and put it at the end: "If you have other feature ideas, ..." ?